src/cmd/ksh93/data/builtins.c

```diff
@@ -1502,7 +1508,7 @@ const char sh_optreadonly[] =
 "[+?\breadonly\b is built in to the shell as a declaration command so that "
 	"field splitting and pathname expansion are not performed on "
 	"the arguments. Tilde expansion occurs on \avalue\a.]"
-"[p?Causes the output to be in a form of \breadonly\b commands that can be "
+"[p?Causes the output to be in the form of \breadonly\b commands that can be "
```

Comment:
I think this should stay the way it is. There are multiple such forms: assignment decoupled from `readonly` commands, or `readonly` with assignment but different quoting styles.